common/config/rush/command-line.json

```diff
@@ -0,0 +1,258 @@
+/**

```

Comment:
Is there any hard run about these files going into a folder called common? I ask mostly as for some of the other languages we have been putting similar engineering system type files under an "eng" directory.

Reply:
I think this comment was intended for #2047? :)

Reply:
lol yes it was :) I was wondering where it disappeared to.